config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root 'static_pages#index'
+  root "games#index"
```

Comment:
This adds multiple root paths to the application.  I think we should stay with the static page for the root path since we already have that set up and then just routing for the games index action right now.